CoC.md

```diff
@@ -10,3 +12,16 @@
 8. Крайне нежелательно дублировать один вопрос в нескольких чатах.
 9. Админы чата могут удалить любое сообщение, не соответствующее правилам.
 10. В рабочих чатах лучше придерживаться бизнес-стиля общения т.е не использовать обсцессивную лексику.
+
+## Code of conduct
+
+1. It is not recommended to write many small messages in a row connected with one thought. State your thoughts in one big message. For large messages, it’s easier to track a thread. Otherwise, it can be considered by the administrator as a flood.
```

Comment:
It is not recommended to write multiple small messages in a row about the same matter. Express your thoughts in one larger message. For large messages, it’s easier to keep track of a thread. Otherwise, it could be considered as flood by the administrator .